packages/documentation/src/content/docs/integration/playground/overview.mdx

```diff
@@ -254,7 +264,7 @@ The Examples folder in the Bruno collection includes an [Open Payments](https://
 
 :::note
 
-You have to go through an interaction flow by clicking on the `redirect` link in the grant request response. More information about the interaction flow can be found <LinkOut href ='https://openpayments.dev/introduction/grants/'>here</LinkOut>.
+You have to go through an interaction flow by selecting the `redirect` link in the grant request response. More information about the interaction flow can be found <LinkOut href ='https://openpayments.dev/introduction/grants/'>here</LinkOut>.
```

Comment:
Maybe "...in the grant request's response". Small thing. I'm not sure if that helps clarify anything.